aqua/constants.aqua

```diff
@@ -0,0 +1,3 @@
+const REPLICATION_FACTOR = 1
+const CONSISTENCY_LEVEL = 1
+const DEFAULT_TIMEOUT = 6000
```

Comment:
Also, need documentation on each of the constants here. What is REPLICATION_FACTOR? Is it for writing or reading? What is CONSISTENCY_LEVEL, how does it relate to REPLICATION_FACTOR?

Reply:
yeah, will add doc comments and explain in API Reference